app/src/main/java/io/stanwood/mhwdb/feature/armors/ui/ArmorDetailsFragment.kt

```diff
@@ -62,7 +64,13 @@ class ArmorDetailsFragment : Fragment(), HasSupportFragmentInjector {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?) =
-        FragmentArmorDetailsBinding.inflate(inflater, container, false, dataBindingComponent)
+        DataBindingUtil.inflate<FragmentArmorDetailsBinding>(
```

Comment:
Is there an advantage in using `DataBindingUtil.inflate()` compared to `*Binding.inflate()`?

Reply:
This usage is deprecated